upgrader_test.go

```diff
@@ -24,8 +24,8 @@ func TestUpgradeHandoff(t *testing.T) {
 
 	// Server 1 starts listening
 	upg1, s1 := createTestServer(t, 1, coordDir, server1Reqs, server1Msgs)
-	// defer upg1.Stop() // TODO, this hangs :(
 	defer s1.Close()
+	defer upg1.Stop()
```

Comment:
Curious how the ordering of this call relates to the hang you saw. Mind describing the original cause?

Reply:
At this point the comment was just stale. When the test was originally being written it hung, I think due to me not having fixed 44df6266bd4c2e7237a044e18e1b55c8d8a2dc48 yet, but the test was committed after that fix (and others), so I think the comment was already stale from the moment it was checked in.